KomodoPlatform/atomicDEX-Desktop
# add KIP0001, update app version

- Adds `KIP0001` as a default coin (wallet only)
- Updates version to `0.5.7.1`

To test:
- Log in to existing wallet. KIP0001 should be activated in your portfolio.
- Create new wallet. KIP0001 should be automatically activated.
- Confirm version number is correct in settings.
- Ping @smk to receive a tiny amount of KIP0001. Confirm balance / history displays as expected. Send funds to one of the vote addresses: `RReduceRewardsXXXXXXXXXXXXXXUxPxuC` or `RKeepRewardsXXXXXXXXXXXXXXXXYKRSuF` 
- Confirm block explorer link works (should go to https://kip0001.explorer.kmd.io/)
- Confirm KIP0001 does not display in the DEX pages.
- Confirm swap button on KIP0001 wallet page is disabled.
